docs/Getting Started/Adding the Lock Screen/index.md

```diff
@@ -23,29 +49,63 @@ SDLLockScreenConfiguration *lockScreenConfiguration = [SDLLockScreenConfiguratio
 ```swift
 let lockScreenConfiguration = SDLLockScreenConfiguration.enabled()
 ```
+!@
+
+@![android]
+`TODO Add code example`
+!@
```

Comment:
Remove as the `Builder` object will automatically configure a default lock screen configuration.

```
@![android]
 `TODO Add code example`
 !@
```

Reply:
removed